docs/maintainer_docs.md

```diff
@@ -0,0 +1,58 @@
+# Maintainers
```

Comment:
perhaps call this file 'maintainers.md', 'maintainer_instructions.md' or similar (to avoid the docs suffix - it's in docs/ already!)